src/components/NewWorkspaceModal.ts

```diff
@@ -318,7 +319,17 @@ const NewWorkspaceModal = withDisplayName(
       // This coupling of enhanced bucket logging and billing project may change in the future.
       return Utils.cond(
         [!!workflowImport || !!requireEnhancedBucketLogging, () => !isAzureBillingProject(project)],
-        [!!cloneWorkspace && isAzureWorkspace(cloneWorkspace), () => isAzureBillingProject(project)],
+        [
+          !!cloneWorkspace && isAzureWorkspace(cloneWorkspace),
+          () => {
+            if (!isAzureBillingProject(project)) {
+              return false;
+            }
+            const azureBillingProject = project as AzureBillingProject;
+            const sourceWorkspace = cloneWorkspace as WorkspaceWrapper;
```

Comment:
@msilva-broad is there a better way to do this? We know that `cloneWorkspace` is not undefined because of line 323, but TS gives a warning.

Reply:
I would recommend to convert this code block to not use Utils.cond and just use basic language constructs.  @nawatts added type safety to Utils.cond a few weeks back, but I think it still falls short of the type flow that would help you here if you were using standard language conditionals.

Reply:
Done.